packages/es-dev-server/src/es-dev-server.ts

```diff
@@ -1,8 +1,16 @@
-export * from './command-line-args';
-export * from './config';
-export * from './create-middlewares';
+import { FSWatcher } from 'chokidar';
+import { createMiddlewares as originalCreateMiddlewares } from './create-middlewares';
+import { ParsedConfig } from './config';
+
 export * from './create-server';
 export * from './start-server';
-
+export * from './command-line-args';
+export * from './config';
 export * from './constants';
 export * from './Plugin';
+export * from './utils/MessageChannel';
+
+// Backwards compatibility, the public API expects an array while we are exporting an object now
+export function createMiddlewares(config: ParsedConfig, fileWatcher: FSWatcher) {
```

Comment:
also no return type, not sure if intentional so just pointing it out

Reply:
Intentional, it's inferred from the return statement.